### PR TITLE
URL hits counter added

### PR DESCRIPTION
**Pull request by jatinkatyal13**

### CODE REVIEW

**Comment by jatinkatyal13** [comment]
I tried this but it throws an error:
``` Property 'increment' does not exist on type 'Model<URLAttributes, URLAttributes>'. ```
Working on resolving this!

**Comment by jatinkatyal13** [comment]
@championswimmer required changes made!

**Comment by championswimmer** [comment]
when doing separate things (hits counter vs pagination) please prefer keeping them in separate PRs

File: views/pages/urls/index.hbs [hunk 1]
```diff
@@ -1,5 +1,19 @@
 {{!< center-container }}
 
+<nav aria-label="Page navigation example">
+    <ul class="pagination justify-content-end">
+        <li class="page-item {{#unless pagination.hasPrev}}disabled{{/unless}}">
+            <a class="page-link" href="?page={{ dec pagination.page }}" tabindex="-1">Previous</a>
+        </li>
+        {{#range pagination.pageCount}}
```
Comment (championswimmer) [inline views/pages/urls/index.hbs]: isn't this something we can handle with an each loop ? handlebar already does have looping support

File: views/partials/navbar.hbs [hunk 2]
```diff
@@ -37,7 +37,7 @@
           <a href="https://account.codingblocks.com" class="nav-link">😎 Profile</a>
         </li>
         <li class="nav-item">
-          <a href="https://account.codingblocks.com" class="nav-link">🔓 Logout</a>
```
Comment (championswimmer) [inline views/partials/navbar.hbs]: this should be a separate change as well, as it is unrelated to hit counter and pagination

Comment (championswimmer) [reply views/partials/navbar.hbs]: I have included this separately under your authorship 
https://github.com/coding-blocks/shortlr2-backend/commit/f9fa9fff63f2e51ce178d367655cde1f8aa36b9d

File: views/pages/urls/index.hbs (hunk 1 above)
Comment (jatinkatyal13) [reply views/pages/urls/index.hbs]: Actually rather than iterating over an iterable I wanted a counter loop! Couldn't find the work around with `#each`

**Comment by jatinkatyal13** [comment]
> when doing separate things (hits counter vs pagination) please prefer keeping them in separate PRs

I'll surely take care of this !

**Pull request closed without merging**